server/server.py

```diff
@@ -127,5 +127,27 @@ def get(self):
 
         return deployments, 200
 
+# Update model location
+@api.route('/update_model_location')
+class UpdateModelLocation(Resource):
+    @api.param('model_id', 'The model ID')
+    @api.param('location', 'The new location')
+    def post(self):
+        """
+        Update the model location.
+        """
+        model_id = request.args.get('model_id')
+        location = request.args.get('location')
+
+        if not model_id or not location:
+            return {"error": "Model ID and Location are required"}, 400
+
+        if not location.startswith("http://") and not location.startswith("https://"):
```

Comment:
This should be a more readable. Refactor this. 
Maybe use urllib.parse instead.

Reply:
Checked:
<img width="900" alt="image" src="https://github.com/user-attachments/assets/946b2efe-f13a-4bd8-94d6-b129429d1c42">

<img width="900" alt="image" src="https://github.com/user-attachments/assets/e095bd7f-c604-49ce-a562-a9b6c7b35b72">